libmscore/text.cpp

```diff
@@ -2649,8 +2649,8 @@ Element* Text::drop(const DropData& data)
                   delete e;
 
                   if (_editMode) {
-                        if (code & 0xffff0000)
-                              insert(_cursor, QChar::highSurrogate(code),  QChar::lowSurrogate(code));
+                        if (QChar::requiresSurrogates(code))
```

Comment:
these two lines are equivalent, but I'm using the Qt Function so that the code is a little clearer.